src/db/index.ts

```diff
@@ -76,26 +75,21 @@ export async function findHostIssues (knex: Knex, host_id: string) {
     .where(RemediationIssueSystems.system_id, EQ, host_id);
 }
 
-function createUpdateQuery (host_id: string, issue_id: string, update: string) {
-    const updateQuery = `UPDATE remediation_issue_systems SET resolved = ${update} ` +
+function createUpdateQuery (host_id: string, issues: string[]) {
+    const formatedIssues = `'` + _.join(issues, `', '`) + `'`;
+    const updateQuery = `UPDATE remediation_issue_systems SET resolved = remediation_issues.issue_id ` +
+    `NOT IN (${formatedIssues}) ` +
```

Comment:
Inserting inputs like this makes the consumer vulnerable to SQL injection. It would be better to use parameter bindings (which sanitize the inputs). http://knexjs.org/#Raw